### PR TITLE
{cae}[dummy/dummy] FLUENT v18.2

### PR DESCRIPTION
**Pull request by shahzebsiddiqui**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/f/FLUENT/FLUENT-18.2.eb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+homepage = 'http://www.ansys.com/Products/Simulation+Technology/Fluid+Dynamics/Fluid+Dynamics+Products/ANSYS+Fluent'
+description = """ANSYS FLUENT software contains the broad physical modeling capabilities needed
+to model flow, turbulence, heat transfer, and reactions for industrial applications ranging from
+air flow over an aircraft wing to combustion in a furnace, from bubble columns to oil platforms,
+from blood flow to semiconductor manufacturing, and from clean room design to wastewater treatment plants."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['FLUIDS_%(version_major)s%(version_minor)s_LINX64.tar']
```
Comment (boegel) [inline easybuild/easyconfigs/f/FLUENT/FLUENT-18.2.eb]: @shahzebsiddiqui The existing easyconfig for `FLUENT` 18.1 uses `FLUIDSTRUCTURES_%(version_major)s%(version_minor)s_LINX64.tar` instead, any idea what the difference is between `FLUIDSTRUCTURES_*.tar` and `FLUIDS_*.tar`?

**Comment by shahzebsiddiqui** [comment]
I have no idea why they changed the name, but when I download FLUENT from
their site the installer file was FLUIDS_182.tar

I suspect there may be some changes, I am not familiar with FLUENT. The
installation work with the current easyblock but I can't be sure if it's
configured properly.

**Comment by boegel** [comment]
@shahzebsiddiqui It seems like the academic version of FLUENT is provided via `FLUIDSTRUCTURES_*_LINX64.tar`, while the commercial version (which I assume is the one you are using) is named `FLUIDS_*_LINX64.tar`.

This poses an interesting problem, not sure how to handle this...

**Comment by shahzebsiddiqui** [comment]
How about using versionsuffix in easyconfig file with word like
StudentEdition and Commercial to make the distinction.

**Comment by boegel** [comment]
@shahzebsiddiqui Using a `versionsuffix` makes sense, but it should be a short one, preferably...

And maybe we should keep using an empty `versionsuffix` for the academic version as we have been doing up until now?

**Comment by migueldiascosta** [comment]
@shahzebsiddiqui ping?

**Comment by boegel** [comment]
@shahzebsiddiqui I think we need to discriminate with the academic version, and I see no other way than adding a versionsuffix, something like `-com` makes sense to me...

**Comment by shahzebsiddiqui** [comment]
@boegel easyconfig updated. Check now

**Pull request merged**